pkg/controller/direct/documentai/processorversion_fuzzer.go

```diff
@@ -29,8 +29,8 @@ func init() {
 
 func DocumentAIProcessorVersionFuzzer() fuzztesting.KRMFuzzer {
```

Comment:
Do we have a processor_fuzzer ?

Reply:
Processor is not ready to promote (no controller). So this PR is to prep processor version for promotion.